src/test/java/org/fcrepo/client/integration/FcrepoClientIT.java

```diff
@@ -511,6 +536,36 @@ public void testHead() throws Exception {
         assertNotNull(headResp.getHeaderValue("Allow"));
     }
 
+    @Test
+    public void testHeadWantDigest() throws Exception {
+        // Creating a binary for retrieval
+        final String mimetype = "text/plain";
+        final String bodyContent = "Hello world";
+        final FcrepoResponse response = client.post(new URI(serverAddress))
+                .body(new ByteArrayInputStream(bodyContent.getBytes()), mimetype)
+                .perform();
+
+        final URI url = response.getLocation();
+
+        final Map<String, Double> qualityMap = new HashMap<>();
+        qualityMap.put("md5", 1.0);
+        qualityMap.put("sha", 1.0);
+        qualityMap.put("sha-256", 0.4);
+
+        final String wantDigest = HeaderHelpers.formatQualityValues(qualityMap);
+        // Get the content of the object after the first 6 bytes
```

Comment:
"after the first 6 bytes"?
Can you explain?

Reply:
Same as above